README.md

```diff
@@ -5,7 +5,7 @@
 
 <p align="center">
 <img src="https://travis-ci.org/MasoniteFramework/masonite.svg?branch=master">
-<img src="https://img.shields.io/badge/python-3.4+-blue.svg" alt="Python Version"> <img src="http://pepy.tech/badge/masonite?1" alt="License">  <img src="https://img.shields.io/github/license/MasoniteFramework/masonite.svg" alt="License"> 
+<img src="<img src="https://img.shields.io/badge/python-3.5+-blue.svg" alt="Python Version">" alt="Python Version"> <img src="http://pepy.tech/badge/masonite?1" alt="License">  <img src="https://img.shields.io/github/license/MasoniteFramework/masonite.svg" alt="License"> 
```

Comment:
This line is technically all screwed up but don't worry its a small change. I merged it in and fixed it @vaibhavmule @danlove99 .

Notice the line actually reads:

```
<img src="<img src="https://img.shields.io/badge/python-3.5+-blue.svg" alt="Python Version">" alt="Python Version">
```

The whole img is inside the src attribute.